controllers/Job/addJob.js

```diff
@@ -0,0 +1,33 @@
+import jobPostings from '../../models/jobPostings.js';
+import { v4 as uuidv4 } from 'uuid';
+
+const addJob = async (req, res) => {
+    const { title, jobType, location, salary, description, requirements, applications,companyName,employerId } = req.body;
+
+    console.log("Received Data on Backend:", req.body);
+    const existingJob = await jobPostings.findOne({ title, employerId });
+    if (existingJob) {
+        return res.status(409).json({ message: "Job with this title already exists for this employer." });
+    }
+    const job = new jobPostings({
+        jobId: uuidv4(),
+        title,
```

Comment:
We can have a status field to keep tack if this is open, closed actively looking.

Also do we need an opening & closing dates?